src/aleph_client/asynchronous.py

```diff
@@ -296,26 +296,31 @@ async def create_store(
 
 
 async def create_program(
-    account: Account,
-    program_ref: str,
-    entrypoint: str,
-    runtime: str,
-    environment_variables: Optional[Dict[str, str]] = None,
-    storage_engine: StorageEnum = StorageEnum.storage,
-    channel: str = settings.DEFAULT_CHANNEL,
-    address: Optional[str] = settings.ADDRESS_TO_USE,
-    session: Optional[ClientSession] = None,
-    api_server: str = settings.API_HOST,
-    memory: int = settings.DEFAULT_VM_MEMORY,
-    vcpus: int = settings.DEFAULT_VM_VCPUS,
-    timeout_seconds: float = settings.DEFAULT_VM_TIMEOUT,
-    persistent: bool = False,
-    encoding: Encoding = Encoding.zip,
-    volumes: Optional[List[Dict]] = None,
-    subscriptions: Optional[List[Dict]] = None,
+        account: Account,
+        program_ref: str,
+        entrypoint: str,
+        runtime: str,
+        allow_amend: bool,
+        use_latest: bool,
+        reproducible: bool,
+        internet: bool,
+        aleph_api: bool,
```

Comment:
You could assign default values here the same as the hardcoded ones before:
```suggestion
        aleph_api: bool = True,
```